packages/client/src/reexports/polkadot-sdk-compat.ts

```diff
@@ -1,2 +1 @@
 export * from "@polkadot-api/polkadot-sdk-compat"
-export { default } from "@polkadot-api/polkadot-sdk-compat"
```

Comment:
This shouldn't be removed (yet) 😉

Reply:
this is a re-export autogenerated by the sync-packages script.... we probably have to improve this

Reply:
It shouldn't be necessary anymore after #614